net/unbound/files/rootzone.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/sh
+##############################################################################
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# Copyright (C) 2016 Eric Luehrsen
+#
+##############################################################################
+#
+# This component needs to be used with the unbound.sh as an include. It uses
+# defaults and UCI scope variables define there. Function rootzone_update() is
+# intended to be part of a crontab or service stop to copy RFC5011 results
+# back to /etc/unbound/root.key.
+#
+##############################################################################
+
+rootzone_uci() {
+  # TODO: place holder; rootzone may be called out of context, so load few UCI
+  echo
+}
+
+##############################################################################
+
+roothints_update() {
+  # TODO: place holder; maybe we won't do this, but structure it in
+  echo
+}
+
+##############################################################################
+
+rootkey_update() {
+  if [ "$UNBOUND_N_ROOT_AGE" -lt 1 \
+      -o "$UNBOUND_N_ROOT_AGE" -gt 90 \
```

Comment:
This will be in the UCI that the user can control how often root.key is copied back to flash ROM. Its just an artifact that shouldn't be at this level.